docs/charts/services-bc-mp/index.yaml

```diff
@@ -2,95 +2,202 @@ apiVersion: v1
 entries:
```

Comment:
Please add `sources:` lists to all the child component charts (pointing to the correlated `microprofile` branches of the individual repositories) so that they will show up here and we can associate tgz files with remote source code.  This does not have to be done in this submission (as it does require updates to all the other repositories), but just wanted it noted here.  I will open another issue to track this.